src/pages/Cart/index.tsx

```diff
@@ -32,8 +33,12 @@ const Cart = () => {
   };
 
   const handleDelete = (id: string) => {
-    removeFromCart(id);
-    addToast(MESSAGE_SUCCESS.DELETE_CART, true);
+    setShowOverlay(true);
+    setTimeout(() => {
```

Comment:
why need to set timeout, any comment here?

Reply:
Set the timeout so that when you delete it, it will overlay and then delete it. Otherwise, it will be hidden without anything happening.